docs/undocumented.json

```diff
@@ -1,12 +1,47 @@
 {
   "warnings": [
```

Comment:
Can we add these missing docs to get 100%?

(or add `/// :nodoc:`)

Reply:
On it